server-spi/src/main/java/org/keycloak/models/ClientPolicyModel.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.models;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class ClientPolicyModel implements Serializable {
```

Comment:
This class (and also ClientProfileModel class) don't need to be here in the "keycloak-server-spi" module. Also those model classes are not typical "model" classes (they are not supposed to be CRUD from DB directly).

So to be able to change them in the future, I vote for move them into "keycloak-server-spi-private" module possibly to the package "org.keycloak.services.clientpolicy" to have most of the client policies stuff in the single package. WDYT?

Reply:
@mposolda Yes, I agree with you. I will try to move `ClientProfileModel` and `ClientPolicyModel` in `org.keycloak.models` of `keycloak-server-spi` project to `org.keycloak.services.clientpolicy`  package of `keycloak-server-spi-private` project.

Reply:
Thanks!